### PR TITLE
Added the ability to define the IndexStyleSheet setting for a directory

### PR DESCRIPTION
**Pull request by genebean**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1948,6 +1948,23 @@ Sets the [default ordering](http://httpd.apache.org/docs/current/mod/mod_autoind
     }
 ```
 
+######`index_style_sheet`
+
+Sets the [IndexStyleSheet](http://httpd.apache.org/docs/current/mod/mod_autoindex.html#indexstylesheet) whcih adds a CSS stylesheet to the directory index.
```
Comment (mhaskel) [inline README.md]: typo on which.

**Comment by mhaskel** [comment]
Hi  @genebean thanks for the contribution! Could you also update the unit tests [here](https://github.com/puppetlabs/puppetlabs-apache/blob/master/spec/defines/vhost_spec.rb) to make sure you can set the new option?

**Comment by genebean** [comment]
@mhaskel - Test added. I also added some for the other parts of my example (which got updated too). I figured they would be needed but couldn't figure out where to put them before you linked the right file... thanks!

**Comment by mhaskel** [comment]
Looks good, thank you!

**Pull request merged**